elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/AggregationDataStoreIntegrationTest.java

```diff
@@ -502,10 +506,298 @@ public void timeGrainAggregationTest() throws Exception {
+                                                )
+                                        )
+                                )
+                        )
+                )
+        ).toQuery();
+
+        String expected = "\"Exception while fetching data (/playerStats) : Invalid operation: 'Can't sort on highScore as it is not present in query'\"";
+
+        runQueryWithExpectedError(graphQLRequest, expected);
+    }
+
+    @Test
+    public void noMetricQueryTest() throws Exception {
+        String graphQLRequest = document(
+                selection(
+                        field(
+                                "playerStats",
+                                selections(
+                                        field(
+                                                "country",
+                                                selections(
+                                                        field("name")
+                                                )
+                                        )
+                                )
+                        )
+                )
+        ).toQuery();
+
+        String expected = "\"Exception while fetching data (/playerStats) : Invalid operation: 'Must provide at least one metric in query'\"";
+
+        runQueryWithExpectedError(graphQLRequest, expected);
+    }
+
+    @Test
+    public void sortingMultipleLevelNesting() throws Exception {
+        String graphQLRequest = document(
+                selection(
+                        field(
+                                "playerStats",
+                                arguments(
+                                        argument("sort", "\"country.continent.name\"")
+                                ),
+                                selections(
+                                        field("lowScore"),
+                                        field(
+                                                "country",
+                                                selections(
+                                                        field("name"),
+                                                        field(
+                                                                "continent",
+                                                                selections(
+                                                                        field("name")
+                                                                )
+                                                        )
+                                                )
+                                        )
+                                )
+                        )
+                )
+        ).toQuery();
+
+        String expected = "\"Exception while fetching data (/playerStats) : Currently sorting on double nested fields is not supported\"";
+
+        runQueryWithExpectedError(graphQLRequest, expected);
+    }
+
+    @Test
+    public void ambiguousFieldTest() throws Exception {
```

Comment:
What is this trying to test?

Reply:
Previously, when we join to two different relationship fields which have the same type, the join would cause ambiguity as the two joined table share the same alias.

Reply:
This kind of test would be clearer as a Unit test that verifies the SQL that gets generated by the ambiguity.

Reply:
👍